tests/modules/services/trayer/basic-configuration.nix

```diff
@@ -0,0 +1,26 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.trayer = {
+      enable = true;
+      package = pkgs.writeScriptBin "dummy-trayer" "";
+      config = {
+        edge = "top";
+        padding = 6;
+        SetDockType = true;
+        tint = "0x282c34";
+        SetPartialStrut = true;
+        expand = true;
+        monitor = 1;
+      };
+    };
+
+    nmt.script = ''
+      serviceFile=home-files/.config/systemd/user/trayer.service
+
+      assertFileExists $serviceFile
+      assertFileRegex $serviceFile 'ExecStart=.*/bin/trayer --SetDockType true --SetPartialStrut true --edge top --expand true --monitor 1 --padding 6 --tint 0x282c34'
```

Comment:
we should also assert the file contents of the config file

Reply:
The config file will be removed.

Reply:
```suggestion
      assertFileContent $serviceFile <<<'ExecStart=@trayer@/bin/trayer --SetDockType true --SetPartialStrut true --edge top --expand true --monitor 1 --padding 6 --tint 0x282c34'
```